frontend/src/general/dialog/DialogBody.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import PropTypes from "prop-types";
+import MuiDialogContent from "@material-ui/core/DialogContent";
+import MuiDialogContentText from "@material-ui/core/DialogContentText";
+
+export default function DialogBody(props) {
+  const { children } = props;
+  return (
+    <MuiDialogContent>
+      <MuiDialogContentText id="dialog-description">
+        {children}
+      </MuiDialogContentText>
+    </MuiDialogContent>
+  );
+}
```

Comment:
Do we really need a component for this? Can we just put the `<MuiDialogContent>` and `<MuiDialogContentText>` directly in the `Dialog` element since there is no other logic here?

Reply:
I prefer having it in its own component because it allows our dialog element to be cleaner, requiring only one tag pair instead of two

Reply:
it's nice to have the test files smaller as well ^